htdocs/js/ui/notebook_protection_controller.js

```diff
@@ -0,0 +1,620 @@
+
+
+define(['angular'], function(angular){
+
+   'use strict';
+
+   return angular.module('myapp.controllers', ['myapp.services', 'selectize'])
+   .controller('NotebookProtectionController', ['$scope' , 'GroupsService', function ($scope, GroupsService){
+
+
+   		//Notebook tab vars
+   		///////////////////////////////
+
+	   	//user 
+	   	$scope.userLogin = '';
+		$scope.userId = '';
+	    
```

Comment:
another nit: please use spaces not tabs. tabs display very differently on different computers.